docs/managed-datahub/observe/assertions.md

```diff
@@ -20,26 +20,35 @@ For DataHub-provided assertion runners, we can deploy an agent in your environme
 - [Custom SQL](/docs/managed-datahub/observe/custom-sql-assertions.md)
 - [Column](/docs/managed-datahub/observe/column-assertions.md)
 
-These can be defined through the DataHub API or the UI.
+**Anomaly detection**
+There are many cases where either you do not have the time to figure out what a good rule for an assertion is, or strict rules do not suffice. For instance, a table whose row count changes exhibit weekly seasonality may need a different set of assertions for each day of the week. Or, figuring out what the expected STD Dev is for each column can be incredibly time consuming and not feasible across hundreds of tables. In these scenarios, you may want to consider creating a [Smart Assertion](./smart-assertions.md) to let ML automatically detect the normal patterns in your data and alert you when anomalies occur.
```

Comment:
I'd break into multiple paragraphs. We can use AI to break it down a bit.

Reply:
See https://github.com/datahub-project/datahub/pull/13782/commits/a80e9145cca2dcd95112cc02029c9d311225f37a